package.json

```diff
@@ -57,7 +57,7 @@
     "dist"
   ],
   "scripts": {
-    "build": "rm -rf dist && babel --extensions '.ts,.tsx' lib/ --out-dir dist/lib && babel tailwind.config.js --out-dir dist && npm run compile:css && npm run legacy:copy-styles && npm run generate-types",
```

Comment:
Is there a reason we're removing the type generation?
I could be wrong, but I think we import those types in SSR so we can have type safety where we're using the mono components?

Reply:
Oops, didn't mean to commit that.

Reply:
Ah no worries :) all good!